rj_gameplay/rj_gameplay/gameplay_node.py

```diff
@@ -65,25 +81,26 @@ def __init__(self, play_selector: situation.IPlaySelector, world_state: Optional
                                                       self.create_goalie_id,
                                                       keep_latest)
 
-        self.robot_state_subs = [None] * NUM_ROBOTS
-        self.robot_intent_pubs = [None] * NUM_ROBOTS
-
-        self.override_actions = [None] * NUM_ROBOTS
+        self.override_actions: List[Optional[IAction]] = [None] * NUM_ROBOTS
 
+        # create lists for robot state subs and robot intent pubs (ROS)
+        self.robot_state_subs = []
         for i in range(NUM_ROBOTS):
-            self.robot_state_subs[i] = self.create_subscription(
+            self.robot_state_subs.append(self.create_subscription(
                 msg.RobotStatus, 'radio/robot_status/robot_' + str(i),
-                self.create_partial_robots, 10)
+                self.create_partial_robots, 10))
 
+        self.robot_intent_pubs = []
         for i in range(NUM_ROBOTS):
-            self.robot_intent_pubs[i] = self.create_publisher(
-                msg.RobotIntent, 'gameplay/robot_intent/robot_' + str(i), 10)
+            self.robot_intent_pubs.append(self.create_publisher(
+                msg.RobotIntent, 'gameplay/robot_intent/robot_' + str(i), 10))
```

Comment:
```
Error: rj_gameplay/rj_gameplay/gameplay_node.py:187: error: "None" has no attribute "publish"
Error: rj_gameplay/rj_gameplay/gameplay_node.py:326: error: "None" has no attribute "tick"
Error: rj_gameplay/rj_gameplay/gameplay_node.py:327: error: "None" has no attribute "publish"
```

These errors stemmed from the fact that these lists were initialized to None and then changed during runtime. I made override_actions a list of Optional[IAction] types, and I changed the initialization for robot_state_subs and robot_intent_pubs such that they would never be None.